docs/antrea-proxy.md

```diff
@@ -253,6 +253,26 @@ data:
       proxyLoadBalancerIPs: false
 ```
 
+With the above configuration, AntreaProxy will ignore all external `loadBalancerIP`s.
```

Comment:
I suppose it should be `loadBalancerIPs`.?

Reply:
This is used to be consistent with others in this document. See L240.

Reply:
I think the current version is fine, but you have to be consistent between `loadBalancerIP`s and `LoadBalancerIP`s

Reply:
Done